include/libtransistor/ipc/time.h

```diff
@@ -15,7 +18,9 @@ result_t time_init();
 /**
 * @brief Get the current system time
 */
-result_t time_get_current_time(uint64_t *time);
+result_t time_systemclock_get_current_time(ipc_object_t systemclock, uint64_t *time);
+result_t time_systemclock_set_current_time(ipc_object_t systemclock, uint64_t *time);
+//result_t time_get_current_time(uint64_t *time);
```

Comment:
Could you remove the comments please ?

Reply:
Sure, will do. Shall I re-add the time_get_current_time function as a shorthand for time_systemclock_get_current_time(time_systemclock_user) or just remove it?

Reply:
nah just remove it. We don't need to bend over for backwards compat yet :P